src/components/auth/profile/DonationTab.tsx

```diff
@@ -8,198 +12,291 @@ import TableBody from '@mui/material/TableBody'
+                  <Typography>Я, Ф, М, А 2022</Typography>
+                </Box>
+                <Typography fontWeight="medium" variant="h5">
+                  {money(userDonations.donations[0].amount)}
+                </Typography>
+              </Box>
+              <Box className={classes.donationsBoxRow}>
+                <Typography variant="h5">{t('auth:profile.donations.cardDonations')}</Typography>
+                <Typography fontWeight="medium" variant="h5">
+                  {money(userDonations.total)}
+                </Typography>
+              </Box>
+              <Box className={classes.donationsBoxRow}>
+                <Typography variant="h5">{t('auth:profile.donations.bankDonations')}</Typography>
+                <Typography fontWeight="medium" variant="h5">
+                  {money(userDonations.total)}
+                </Typography>
+              </Box>
+            </Card>
+          ) : (
+            <CircularProgress />
+          )}
+        </Grid>
+        <Grid order={matches ? 2 : 3} item xs={12}>
+          <Card sx={{ padding: theme.spacing(2) }}>
+            <Grid container alignItems={'flex-start'} spacing={theme.spacing(2)}>
+              <Grid item xs={6} sm={3}>
+                <CheckboxLabel>{t('auth:profile.donations.oneTime')}</CheckboxLabel>
+                <Checkbox
+                  onChange={(e, checked) => {
+                    setOneTime(checked)
+                  }}
+                  checked={oneTime}
+                  name="oneTime"
+                />
+              </Grid>
+              <Grid item xs={6} sm={3}>
+                <CheckboxLabel>{t('auth:profile.donations.monthly')}</CheckboxLabel>
+                <Checkbox
+                  onChange={(e, checked) => {
+                    setMonthly(checked)
+                  }}
+                  checked={monthly}
+                  name="monthly"
+                />
+              </Grid>
+              <LocalizationProvider
+                locale={i18n.language === 'bg' ? bg : enUS}
+                dateAdapter={AdapterDateFns}>
+                <Grid item xs={12} sm={3}>
+                  <DatePicker
+                    label={t('auth:profile.donations.fromDate')}
+                    value={fromDate}
+                    onChange={setFromDate}
+                    renderInput={(params) => <TextField size="small" {...params} />}
+                  />
+                </Grid>
+                <Grid item xs={12} sm={3}>
+                  <DatePicker
+                    label={t('auth:profile.donations.toDate')}
+                    value={toDate}
+                    onChange={setToDate}
+                    renderInput={(params) => <TextField size="small" {...params} />}
+                  />
+                </Grid>
+              </LocalizationProvider>
+            </Grid>
+            {filteredDonations?.length ? (
+              <TableContainer>
```

Comment:
Let's extract this table in a separate component `DonationTable` and pass the donations to it

```tsx
<DonationTable donations={filteredDonations} />
```

Reply:
Done in https://github.com/podkrepi-bg/frontend/pull/719/commits/15d0d06b172ace6de860f1dc6336566ff4dad008